### PR TITLE
UIU-2423 - create Jest/RTL test for NoteCreatePage.js

### PR DESCRIPTION
**Pull request by OybekMamatovEPAMSystems**
Issue : [UIU-2423](https://issues.folio.org/browse/UIU-2423)
create Jest/RTL test for NoteCreatePage.js

### CODE REVIEW

**Review by Godlevskyi — APPROVED** [review]

File: src/views/Notes/NoteCreatePage.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <NoteCreatePage {...props} />
+    Note Create Page
+  </div>
+);
+
+describe('Note Create Page', () => {
+  it('should render Note Create Page', async () => {
+    const { getByText } = renderNoteCreatePage();
+
+    expect(getByText('Note Create Page')).toBeInTheDocument();
```
Comment (mkuklis) [inline src/views/Notes/NoteCreatePage.test.js:26]: @OybekMamatovEPAMSystems it looks like we are testing something which is provided in `renderNoteCreatePage ` above but not really anything related to `<NoteCreatePage />`. I don't think this is needed and we should instead try to test `NoteCreatePage`.

**Review by mkuklis — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -52,6 +53,7 @@
 * create Jest/RTL test for src/components/Accounts/Filters. Refs UIU-2356.
 * Handle time zone issues in user info date displays. Fixes UIU-2623.
 * User should be directed to the last open page when working in more than one browser. Refs UIU-2605.
+* create Jest/RTL test for NoteCreatePage.js. Refs UIU-2423.
```
Comment (NikitaSedyx) [inline CHANGELOG.md:56]: this line is duplicated

File: src/views/Notes/NoteCreatePage.test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+const renderNoteCreateRoute = (props = {
+  history,
+  location
+}) => render(
+  <NoteCreateRoute {...props} />
+);
+
+describe('Note Create Page', () => {
+  it('should render Note Create Page', async () => {
```
Comment (NikitaSedyx) [inline src/views/Notes/NoteCreatePage.test.js:35]: could you please introduce one more test case for scenario when location.state is defined?

Comment (NikitaSedyx) [reply src/views/Notes/NoteCreatePage.test.js:35]: also would be nice to test what passed to referredEntityData prop when state defined and not

**Pull request merged**